tests/src/test/java/alluxio/client/fs/concurrent/ConcurrentFileSystemMasterSetTtlIntegrationTest.java

```diff
@@ -173,4 +177,93 @@ private void assertErrorsSizeEquals(ConcurrentHashSet<Throwable> errors, int exp
           expected, errors.size()) + Joiner.on("\n").join(errors));
     }
   }
+
+  @Test
+  public void testConcurrentInsertAndExpire() throws Exception {
+    /* (ALLUXIO-2821) When an inode is concurrently added to ttlbucket and inodettlchecker
```

Comment:
nit: remove ticket number if applicable

Reply:
? this is specifically to test the fix that got this ticket addressed

Reply:
discussed offline, this is to specifically test the fix to address this ticket, will modify the comment